examples/streaming_generator_example.py

```diff
@@ -1,14 +1,14 @@
 import os
 from salesgpt.agents import SalesGPT
 
-from langchain.chat_models import ChatOpenAI
+from langchain.chat_models import ChatLiteLLM
 
 with open('.env','r') as f:
     env_file = f.readlines()
 envs_dict = {key.strip("'") :value.strip("\n") for key, value in [(i.split('=')) for i in env_file]}
 os.environ['OPENAI_API_KEY'] = envs_dict['OPENAI_API_KEY']
 
-llm = ChatOpenAI(temperature=0.9)
+llm = ChatLiteLLM(temperature=0.9)
```

Comment:
Does this library support `https://github.com/filip-michalsky/SalesGPT/blob/main/salesgpt/agents.py#L139`?

we are leveraging `completion_with_retry` from core langchain.